### ISSUE TITLE
Cannot sync forms pushed to KC by ODK Briefcase

### ISSUE THREAD

**Issue by jnm** [kobotoolbox/kpi#1251]
When Briefcase pushes forms to KC, it does so by uploading XML only. KPI expects each form to have an XLS representation stored.

https://www.flowdock.com/app/kobotoolbox/kobo/threads/u8ZQ5dI6zdViV0wKNofHbqBnHx1
https://www.flowdock.com/app/kobotoolbox/support/threads/qQay5ZLV9lJbY4ZdSOW6IemdP6c

**Comment by jnm** [comment]
Possible workaround: get users to upload XLSForm to KC and then push submissions with Briefcase?

**Comment by jnm** [comment]
Is there an easy way to go from XForm XML to XLSForm?

**Comment by tinok** [comment]
> Is there an easy way to go from XForm XML to XLSForm?

I don't think so, no. We built a rudimentary XML importer into `dkobo` back in 2014 to allow importing old XML-only forms (only question structure and labels, no settings). I think deploying the XLSForm first should be the prerequisite to get this working. Just need to make this explicit in our help pages.
